patches/tModLoader/Terraria/ModLoader/Default/UnloadedWallInfo.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Terraria.ModLoader.IO;
+
+namespace Terraria.ModLoader.Default
+{
+	internal class UnloadedWallInfo : UnloadedInfo 
+	{
+		
+		public UnloadedWallInfo(string modName, string name, ushort fallbackType = 0) :base(modName, name,fallbackType) {
```

Comment:
```suggestion
	{
		public UnloadedWallInfo(string modName, string name, ushort fallbackType = 0) :base(modName, name,fallbackType) {
```

Reply:
Add a space between `name,` and `fallbackType`.

Reply:
Resolved spacing issues in all info subclasses.